third_party/mkl_dnn/mkldnn_v1.BUILD

```diff
@@ -116,7 +116,6 @@ _COPTS_LIST = select({
     "-UUSE_MKL",
     "-UUSE_CBLAS",
     "-DDNNL_ENABLE_MAX_CPU_ISA",
-    "-DDNNL_DISABLE_PRIMITIVE_CACHE",
```

Comment:
@ashiqimranintel This is intentional, right?

Reply:
hi @penpornk yes.  This reverts a change in 2.10 https://github.com/tensorflow/tensorflow/pull/57071 due to a mem corruption that is now fixed.

Reply:
@agramesh1 Thank you for the quick reply!